### PR TITLE
pascal ojinnaka

### PR DESCRIPTION
**Pull request by passy4ucj**

### CODE REVIEW

File: data.json [hunk 1]
```diff
@@ -532,5 +532,12 @@
...
     "imageUrl": "https://user-images.githubusercontent.com/31401437/106563296-974b8f80-6551-11eb-8753-a13ea9e944c2.png",
     "issueNumber": 304
+  },
+  {
+    "name": "Pascal Ojinnaka",
+    "githubUsername": "passy4ucj",
+    "imageUrl": "https://user-images.githubusercontent.com/56399504/107071622-46ec6000-67e5-11eb-8ce8-dc2b06f16dae.PNG",
+    "issueNumber": 305
   }
+
```
Comment (AllanRegush) [inline data.json:542]: I don't think we need this new line here :nerd_face:

File: data.json [hunk 2]
```diff
@@ -532,5 +532,12 @@
     "githubUsername": "adityaoberai",
     "imageUrl": "https://user-images.githubusercontent.com/31401437/106563296-974b8f80-6551-11eb-8753-a13ea9e944c2.png",
     "issueNumber": 304
+  },
+  {
+    "name": "Pascal Ojinnaka",
+    "githubUsername": "passy4ucj",
+    "imageUrl": "https://user-images.githubusercontent.com/56399504/107071622-46ec6000-67e5-11eb-8ce8-dc2b06f16dae.PNG",
+    "issueNumber": 305
```
Comment (AllanRegush) [inline data.json:540]: #305 is a pull request. Did you mean to add #336?

**Review by AllanRegush — COMMENTED** [review]
Looking Good :+1: Just some minor adjustments. :smile:

File: data.json (hunk 2 above)
Comment (passy4ucj) [reply data.json:540]: Yes. I think it's a typo. Thank you

**Review by AllanRegush — APPROVED** [review]
Looks Good :+1: Thank You :smile:

**Comment by AllanRegush** [comment]
It looks like there is a conflict Could you fix this? If you need any help let me know :nerd_face:

**Pull request merged**